ui/static/ui/js/listing.js

```diff
@@ -213,6 +229,72 @@ define(
         });
       });
 
+      var updateCheckDisplay = function(node) {
+        if ($(node).data("selected")) {
+          $(node).find("i").attr("class", "fa fa-check");
+        } else {
+          $(node).find("i").attr("class", "fa fa-arrow-right");
+        }
+      };
+      /**
+       * Update export count badge
+       * @param {number} direction Add this value to
+       * the existing count before rendering
+       */
+      var updateExportCount = function(direction) {
+        var oldCount = $("#export_button").data("export-count");
+        var newCount = oldCount + direction;
+
+        $("#export_button .badge").remove();
```

Comment:
I was referring to building the `<ul>` elements (search for `var exports`), and `errorBox` and `messageBox`. If there's a reason not to use React that's fine; I just wanted to mention it.